omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -362,4 +370,13 @@ def err_unused_postgres_configurable(setting)
       Server. Please check the release notes for details.
     EOM
   end
+
+  def err_unsupported_postgres_version(ver)
+    <<~EOM
+      Chef Server currently supports PostgreSQL version: #{SUPPORTED_VERSION}.
```

Comment:
nit, this shouldn't have a ":" because we're not listing a thing - we're identifying the specific version.